sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/VirtualMachine.cs

```diff
@@ -151,14 +151,16 @@ public VirtualMachine()
+        public VirtualMachine(string location, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), Plan plan = default(Plan), HardwareProfile hardwareProfile = default(HardwareProfile), StorageProfile storageProfile = default(StorageProfile), AdditionalCapabilities additionalCapabilities = default(AdditionalCapabilities), OSProfile osProfile = default(OSProfile), NetworkProfile networkProfile = default(NetworkProfile), SecurityProfile securityProfile = default(SecurityProfile), DiagnosticsProfile diagnosticsProfile = default(DiagnosticsProfile), SubResource availabilitySet = default(SubResource), SubResource virtualMachineScaleSet = default(SubResource), SubResource proximityPlacementGroup = default(SubResource), string priority = default(string), string evictionPolicy = default(string), BillingProfile billingProfile = default(BillingProfile), SubResource host = default(SubResource), SubResource hostGroup = default(SubResource), string provisioningState = default(string), VirtualMachineInstanceView instanceView = default(VirtualMachineInstanceView), string licenseType = default(string), string vmId = default(string), string extensionsTimeBudget = default(string), int? platformFaultDomain = default(int?), ScheduledEventsProfile scheduledEventsProfile = default(ScheduledEventsProfile), IList<VirtualMachineExtension> resources = default(IList<VirtualMachineExtension>), VirtualMachineIdentity identity = default(VirtualMachineIdentity), IList<string> zones = default(IList<string>), ExtendedLocation extendedLocation = default(ExtendedLocation))
```

Comment:
Adjust swagger field order to avoid signature change.

Reply:
@allenjzhang , we have already done our best to do this, but it is not possible.

@shpa-microsoft , you will need to create a [Customization ](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations)file for this issue. 
1 file in the Customization folder for each file in this PR that has a new parameter (not the last parameter) in the method signature.
Each Customization file needs these:
1) 1 method with each parameter up to but not including the new parameter.
2) 1 method for each parameter that is after the new parameter. 
This means 1 new method for each parameter up to the parameter 'resources' NOT including the new ScheduledEvents parameter. 
Then 1 new method for each parameter up to 'identity' NOT including ScheduledEvents.
Do this until you reach the last parameter.